globus_automate_client/cli/helpers.py

```diff
@@ -21,7 +26,9 @@ def format_and_echo(result: Union[GlobusHTTPResponse, str], verbose=False) -> No
         result = result.data
     else:
         color = typer.colors.GREEN
-    typer.secho(json.dumps(result, indent=4, sort_keys=True), fg=color)
+
+    typer.secho(dumper(result, indent=4, sort_keys=True), fg=color)
```

Comment:
Passing the indent/sort_keys to dumper seems a bit dangerous unless we know all possible dumper functions can take those as args. Perhaps the re-usable function/lambda that can be used by callers is a cleaner approach?